test/test_mcmc.py

```diff
@@ -433,7 +433,7 @@ def potential_fn(z):
     rng_keys = random.split(random.PRNGKey(0), 2)
 
     init_kernel_map = map_fn(lambda init_param, rng_key: init_kernel(
-        init_param, trajectory_length=9, num_warmup=warmup_steps, progbar=False, rng_key=rng_key))
+        init_param, trajectory_length=9, num_warmup=warmup_steps, rng_key=rng_key))
```

Comment:
It is strange that this wasn't throwing an error in travis.

Reply:
I think we missed this test when merging two test_mcmc files. Currently, we only test for chain: `pytest -vs test/test_mcmc.py -k chain`.

Reply:
Thanks, @fehiepsi. That was my mistake, fixed.